manage.py

```diff
@@ -46,10 +53,24 @@ def test(all=False):
 def scrape():
     """Start an infinte loop to scrape & categorize articles."""
 
+    q = Queue('default', connection=conn)
     while True:
         logging.info("Starting scrape...")
-        scrape_and_categorize_articles()
-        logging.info("Finished scrape. Let's run it back...")
+        job = q.enqueue(scrape_and_categorize_articles)
```

Comment:
Doesn’t seem necessary. It requires the option ```depends_on```=scrape_job_id, and the job won’t become enqueued until scrape finishes anyway (how rq works), so effectively it’s just the big function anyway. Might as well wait for the big scrape & categorization then.